Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -51,6 +51,14 @@ open class ChatMessageContentView: _View, ThemeProvider {
     /// When this value is set the subviews are instantiated and laid out just once based on
     /// the received options.
     public var layoutOptions: ChatMessageLayoutOptions?
+    
+    /// The formatter used for text Mardown
+    public var markdownFormatter: MarkdownFormatter = DefaultMarkdownFormatter()
```

Comment:
Instead of having 2 boolean flags + `markdownFormatter` in each message content view, I would just have a single optional formatter in `Components`. If it's nil, it means the markdown is disabled. At the same time, user can inject their own formatter if they want.
```swift
public struct Components {
 public var markdownFormatter: MarkdownFormatter? = DefaultMarkdownFormatter()
}
```

Reply:
I get your point @evsaev, but I actually think that having the flag might be easier to understand and easier to find what you need in this case 🤔

Reply:
Keep the bool for consistency as recommended by Nuno.